### ISSUE TITLE
Commercial Use

### ISSUE THREAD

**Issue by Hegelim** [openlm-research/open_llama#20]
Just kindly want to ask is OpenLLaMA eligible for commercial use (both the code and the pre-trained weights)? The Apache-2.0 license seems Ok but I just to confirm it with you guys. Thanks.

**Comment by cemkoc** [comment]
> Just kindly want to ask is OpenLLaMA eligible for commercial use (both the code and the pre-trained weights)? The Apache-2.0 license seems Ok but I just to confirm it with you guys. Thanks. 

Yes it is okay.

**Comment by young-geng** [comment]
> Just kindly want to ask is OpenLLaMA eligible for commercial use (both the code and the pre-trained weights)? The Apache-2.0 license seems Ok but I just to confirm it with you guys. Thanks.

Yes it is definitely okay for commercial use and this is one of the important goals of this project.
